src/stirling/source_connectors/socket_tracer/bcc_bpf/BUILD.bazel

```diff
@@ -51,12 +51,16 @@ cc_library(
 pl_bpf_cc_resource(
     name = "socket_trace",
     src = "socket_trace.c",
+    # Uncomment to enable debug printks.
+    # defines = ["BPF_DEBUG"],
+    defines = select({
+        "@platforms//cpu:aarch64": ["TARGET_ARCH_AARCH64"],
+        "@platforms//cpu:x86_64": ["TARGET_ARCH_X86_64"],
+    }),
```

Comment:
Since BPF is compiled for the bpf target and not x86 or aarch64, I wanted to make a distinction between the architecture being compiled for and the architecture the JIT'd bpf code will eventually run on.